nixos/modules/services/databases/mongodb.nix

```diff
@@ -32,8 +32,7 @@ in
       enable = mkEnableOption "the MongoDB server";
 
       package = mkOption {
-        default = pkgs.mongodb;
```

Comment:
We should keep a default package

Reply:
But this option will be overridden with the condition of stateVersion (a little bit below this snippet), so why define a version here? Also, if defined, won't conflict with this same condition?